GravityParticles.podspec

```diff
@@ -0,0 +1,22 @@
+
+Pod::Spec.new do |s|
+  s.name             = 'GravityParticles'
+  s.version          = '0.3.0'
+  s.source           = {
+    :git => 'https://github.com/buildit/gravity-particles.git',
+    :tag => s.version.to_s
+  }
+
+  s.summary          = 'GravityParticles to be used on iOS when styling your UIKit.'
+  s.description      = 'This library will contain basic swift files using style dictionnary from Amazon'
```

Comment:
I think it would be better to describe what consumers of this CocoaPod get rather than how we produce it.

Maybe something like "This library contains the core, visual design attributes (aka design tokens) defined by the Gravity design system as Swift code."

Reply:
resolving as this is part of the other branch now since we split it into 2 @james-nash 
Agreed and also changed the description into something more meaningful.